public/app/core/components/OptionsUI/registry.tsx

```diff
@@ -81,7 +81,8 @@ export const getAllOptionEditors = () => {
     name: 'Boolean',
     description: 'Allows boolean values input',
     editor(props) {
-      return <Switch {...props} onChange={(e) => props.onChange(e.currentTarget.checked)} />;
+      const { id, ...rest } = props; // Remove id from properties passed into switch
```

Comment:
Not sure on this approach given whether or not we are sure that the `id` prop is not necessary / being used (both now / in future) for testing purposes etc. for the switches in the editor

Reply:
Yeah, feel the same way. Another possible approach would be to update this code to exclude tagName of "INPUT":
https://github.com/grafana/grafana/blob/main/public/app/core/navigation/GrafanaRoute.tsx#L78-L86

Reply:
Talked to Ryan a bit about it, given in testing we shouldn't rely on `id` anyways (and use `aria-label` or data-testid if necessary) this change is probably ok